ipatests/test_integration/test_ntp_options.py

```diff
@@ -0,0 +1,226 @@
+               '-n', self.master.domain.name, '-r', self.master.domain.realm,
+               '--server', self.master.hostname, '-U', *args]
+        return self.replica.run_command(cmd, raiseonerr=False)
+
+    def test_server_and_client_install_without_option_n(self):
+        """
+        test to verify that ipa-server and ipa-client install uses
+        default chrony configuration without any NTP options specified
+        """
+        expected_msg1 = "No SRV records of NTP servers found and " \
+                        "no NTP server or pool address was provided."
+        expected_msg2 = "Using default chrony configuration."
+
+        server_install = tasks.install_master(self.master, setup_dns=False)
+        assert expected_msg1 in server_install.stderr_text
+        assert expected_msg2 in server_install.stdout_text
+
+        client_install = self.install_client()
+        assert expected_msg1 in client_install.stderr_text
+        assert expected_msg2 in client_install.stdout_text
+
+        self.cleanup()
+
+    def test_server_and_client_install_with_option_n(self):
+        """
+        test to verify that ipa-server and ipa-client install with option -N
+        """
+        expected_msg1 = "Excluded by options:"
+        expected_msg2 = "Using default chrony configuration."
+
+        server_install = tasks.install_master(self.master, setup_dns=False,
+                                              extra_args=['-N'])
+        assert expected_msg1 in server_install.stdout_text
+        assert expected_msg2 not in server_install.stdout_text
+
+        client_install = self.install_client('--no-ntp')
+        assert expected_msg2 not in client_install.stdout_text
+
+        self.cleanup()
+
+    def test_server_and_client_install_with_multiple_ntp_server(self):
+        """
+        test to verify that ipa-server-install with multiple --ntp-server
+         option
+        """
+        ntp_server1 = "1.pool.ntp.org"
+        ntp_server2 = "2.pool.ntp.org"
+        expected_msg = "Configuration of chrony was changed by installer."
+
+        server_install = tasks.install_master(
+            self.master, setup_dns=False,
+            extra_args=['--ntp-server=%s' % ntp_server1,
+                        '--ntp-server=%s' % ntp_server2])
+        assert expected_msg in server_install.stderr_text
+        cmd = self.master.run_command(['cat', paths.CHRONY_CONF])
+        assert ntp_server1 in cmd.stdout_text
+        assert ntp_server2 in cmd.stdout_text
+
+        client_install = self.install_client('--ntp-server=%s' % ntp_server1,
+                                             '--ntp-server=%s' % ntp_server2)
+        assert expected_msg in client_install.stderr_text
+        cmd = self.client.run_command(['cat', paths.CHRONY_CONF])
+        assert ntp_server1 in cmd.stdout_text
+        assert ntp_server2 in cmd.stdout_text
+
+        self.cleanup()
+
+    def test_server_replica_and_client_install_with_ntp_pool_and_ntp_server(
```

Comment:
Please rename method to:
```suggestion
    def test_server_replica_client_install_with_pool_and_srv(
```